recipes/baical-p7/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(p7 REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+
+if(P7_SHARED)
+  target_link_libraries(${PROJECT_NAME} p7::p7-shared)
+else()
+  target_link_libraries(${PROJECT_NAME} p7::p7)
+endif()
+
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
```

correct me if i am wrong but I do not see any C++11 features in the test package

```suggestion
endif()
```

---

Done! Somehow I couldn't commit your suggestions.